README.md

```diff
@@ -1 +1,43 @@
-# Recreating HackerNews in Angular
\ No newline at end of file
+# Recreating HackerNews in Angular
+
+## Objectives
+
+- Use the HackerNews API to create a clone app
+- Render the top stories view
+- Use ui-router for advanced routing
+- Use dynamic views in routes
+- Set the top stories view as the default route
+- Create relevant Controllers and Services
+- Use relevant Directives to bind ViewModels and Events
+- Use routing resolves to preload data fetched from Services
+- Allow a user to transition to a news item when clicked
+- Show all comments for a post in the news item
+- Nest replies to comments where necessary
+- Display dates in human-readible formats
+- Allow the user to navigate "Next" and "Previous" page
+
+## Instructions
+
+To run the local dev server, you'll need to run `npm install` and then `node server.js`. This will run a local dev server at http://localhost:8080.
+
+HackerNews is a basic news site where people can post items and comment on them. Let's recreate some of it.
```

Comment:
Include a link to HackerNews here, in case students aren't familiar they can get an idea of what they're building.